cypress/integration/rancid_tomatillos_spec1.js

```diff
@@ -0,0 +1,40 @@
+import movieData from './data.js';
+
+describe('App', () => {
+
+  it('should be able to visit app and render the correct elements', () => {
+    cy.visit('http://localhost:3000')
+    const allMovieTitles = movieData.movies.map(movie => movie.title);
+
+    allMovieTitles.forEach(movieTitle => {
+      cy.get('div').contains(movieTitle)
+    });
+
+    cy.contains('Rancid Tomatillos')
+  });
+
+  it('should show an informative error message when the server is down', () => {
+    cy.intercept({
+        method: 'GET',
+        url: 'https://rancid-tomatillos.herokuapp.com/api/v2/movies'
+      },
+      {
+        statusCode: 422,
+      })
+      cy.visit('http://localhost:3000')
+      .get('h2').should('contain', 'Oh wow. This is embarassing. Try again later? 😅')
```

Comment:
Glad we figured this out together. This test may prevent us from using a beforeEach to simplify the code. Once we have our test files consolidated we can look for any repeating expressions to refactor.